app/scripts/services/brackets.coffee

```diff
@@ -10,11 +10,13 @@
 angular.module('edudashAppSrv').service 'bracketsSrv', ->
 
   getBracket: (val, metric) ->
-    unless typeof val in ['number', 'undefined']
+    unless typeof val in ['number', 'undefined', 'string']
       throw new Error "val must be a number. Got: '#{val}' which is '#{typeof val}'"
     if isNaN val  # NaN or undefined
       'UNKNOWN'
     else
+      if typeof val is 'string' # this is a hack since the negative come as string values
+        val = parseInt val
```

Comment:
Careful, `parseInt` truncates any floating-point values. `parseFloat` is safer if we are accepting arbitrary strings.

Where are the negative-number strings coming from? I think it would be better to either put this conversion there, or wherever we call `getBracket` that might have a string, instead of in here.